2-advanced-features.ipynb

```diff
@@ -5,13 +5,19 @@
    "id": "f3e6c3f1-9673-4eb0-81b1-3fe9ac7e84cf",
```

Comment:
<p>Change logo src for </p><p><code>"https://raw.githubusercontent.com/dask/dask/main/docs/source/images/dask_horizontal.svg"</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jsignell/dask-tutorial-advanced/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>